### ISSUE TITLE
Avoid redundancies in yaml `fm_` entries

### ISSUE THREAD

**Issue by Flix6x** [SeitaBV/v2g-liberty#60]
There seems overlap in the yaml entries. fm_api and fm_data_api are almost identical.
the api_version is a separate entry and is in the fm_data_api_sensor_data.
Could these be combined or is this redundancy needed (in the future)?

_Originally posted by @ArdJonker in https://github.com/SeitaBV/v2g-liberty/pull/58#pullrequestreview-1228196431_

**Comment by ArdJonker** [comment]
Yes! And together with this i'd like to move a lot of the configuration entries from secrets.yaml and with that from apps.yaml to the constans.py.
This also in preparation for adding the optimisation for specific energy companys which all have a differnt sensor.
